src/python/qeqiskit/simulator/simulator.py

```diff
@@ -44,12 +45,13 @@ def __init__(
         Returns:
             qeqiskit.backend.QiskitSimulator
         """
+        self.number_of_circuits_run = 0
+        self.number_of_jobs_run = 0
+
```

Comment:
Similar to `QiskitBackend`, I think it would be better to use `super().__init__(n_samples=n_samples)`

Reply:
Addressed in [d81040](https://github.com/zapatacomputing/qe-qiskit/pull/18/commits/d8104090971fb60f46de371cafe4e6e745c211d3).